internal/impl/kafka/enterprise/redpanda_migrator_bundle_output.tmpl.yaml

```diff
@@ -62,7 +62,6 @@ mapping: |
       "partition": "${! metadata(\"kafka_partition\").or(throw(\"missing kafka_partition metadata\")) }",
       "partitioner": "manual",
       "timestamp_ms": "${! metadata(\"kafka_timestamp_ms\").or(timestamp_unix_milli()) }",
-      "max_in_flight": this.redpanda_migrator.max_in_flight.or(1),
```

Comment:
I forgot to update this when I switched to the new `FranzReaderOrdered` / `FranzWriter` APIs. The default value (256) should be fine and it should not break message ordering.